docker-compose.yml

```diff
@@ -1,27 +1,37 @@
-db:
-    image: mongo:3.2
-    command: mongod
-    volumes:
-      - ./data/db:/data/db
-    expose:
-      - "27017"
-    ports:
-    - "27017:27017"
+version: "3"
+services:
+    db:
+        image: mongo:3.2
+        command: mongod
+        volumes:
+          - ./data/db:/data/db
+        expose:
+          - "27017"
+        ports:
+        - "27017:27017"
 
-broker:
-    image: redis
-    volumes:
-      - ./data/broker:/data
-    expose:
-      - "6379"
-    ports:
-    - "6379:6379"
+    mongo_test:
+        image: mongo:3.4.0
+        tmpfs: /tmp/db
```

Comment:
Is this the right path ? (ie. I think it should be `tmpfs: /data/db` to be effective)